phpBB/phpbb/di/extension/core.php

```diff
@@ -68,6 +68,19 @@ public function load(array $configs, ContainerBuilder $container)
 			}
 		}
 
+		// Set the Twig options if defined in the environment
+		$definition = $container->getDefinition('template.twig.environment');
+		$twig_environment_options = $definition->getArgument(6);
```

Comment:
I don't think a constant will have any sense (and where do you want to define it?)